build_tools/python/benchmarks/iree/definitions.py

```diff
@@ -45,19 +63,22 @@ def generate(
             compile_config=cls.CASCADELAKE_COMPILE_CONFIG, model=model)
         for model in model_groups.MOBILE
     ]
+    # Generate compile specs for workstation models.
+    workstation_model_compile_specs = [
+        iree_definitions.CompileSpec(
+            compile_config=cls.CASCADELAKE_COMPILE_CONFIG, model=model)
+        for model in model_groups.WORKSTATION
+    ]
+
+    # Generate run specs for all models.
+    compile_specs = (mobile_model_compile_specs +
```

Comment:
These comments seem superfluous. Especially the last one, where you're clearly just adding two lists together (same with the "...mobile..." above

```suggestion
    workstation_model_compile_specs = [
        iree_definitions.CompileSpec(
            compile_config=cls.CASCADELAKE_COMPILE_CONFIG, model=model)
        for model in model_groups.WORKSTATION
    ]

    compile_specs = (mobile_model_compile_specs +
```

Reply:
Realized I can just concat two model lists first... Cleaned up.